ansible/roles/atmo-mount-volume/tasks/main.yml

```diff
@@ -16,7 +16,12 @@
     src: '{{ VOLUME_DEVICE }}'
     state: 'mounted'
     path: '{{ VOLUME_MOUNT_LOCATION }}'
+    passno: '{{ VOLUME_PASSNO }}'
+    opts: '{{ VOLUME_OPTS }}'
   register: mount_result
+  vars:
+    UBUNTU_14: ansible_distribution == 'Ubuntu' and ansible_distribution_major_version < '16'
```

Comment:
Sort an edge case, but you're really capturing anything less than ubuntu 16, like ubuntu 12. Would change to `UBUNTU_16: ansible_distribution == 'Ubuntu' and ansible_distribution_major_version == '16'` and update the ternary.

Reply:
Actually i think my suggestion is bad. I'm okay with how it is,  the change for ubuntu 16 is probably going to extend into ubuntu 18 etc